atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -293,13 +293,22 @@ static String getFilteredStackTrace(String filter) {
     }
 
     static Column createColumn(Cell cell, Value value) {
+        return createColumn(cell, value, value.getTimestamp());
+    }
+
+    static Column createColumn(Cell cell, Value value, long writeTimestamp) {
         byte[] contents = value.getContents();
-        long timestamp = value.getTimestamp();
-        ByteBuffer colName = makeCompositeBuffer(cell.getColumnName(), timestamp);
+        ByteBuffer colName = makeCompositeBuffer(cell.getColumnName(), value.getTimestamp());
```

Comment:
This isn't a bug, but at first glance it looked like one. Might need more specific variable names and/or a comment to make explicit that `writeTimestamp` means `cassandraTimestamp` and `value.getTimestamp()` is `atlasTimestamp`.

If we only intend to supply a `writeTimestamp` when deleting, we could call the method `createColumnForDelete`, and add a javadoc explaining why we use an unusual (for Atlas) cass timestamp.

Reply:
Or do the code changes to this file just disappear once we roll back the changes related to `multiPut`?